### DISCUSSION TITLE
How to enable browser sync?

### DISCUSSION THREAD

**Discussion by flintcreativestudio** [tailpress/tailpress#108] [Q&A] [answered]
Hi there,

Can you kindly provide some instructions on how to enable browser sync or live reload with this package please?

**Comment by jeffreyvr** [comment, marked as answer]
Hi, 

Check the following npm script:

https://github.com/jeffreyvr/tailpress/blob/master/package.json#L28

Make sure to change the domain to the one you are using.

Then run: npm run watch-sync

**Reply by flintcreativestudio** [reply]
Oh it was there all along! How stupid of me. Thank you!

**Reply by justingolden21** [reply]
Hi, where do I change the domain here? 
`"watch-sync": "cross-env NODE_ENV=development concurrently \"npm run browser-sync\" \"npm run watch\""`
Do I need to make an `.env` file for `cross-env` to consume?
Thanks in advance! : )

**Reply by jeffreyvr** [reply]
You'll find the domain in the `browser-sync` script in package.json.

**Reply by charlie-george-1989** [reply]
This was useful for me too. I think it's this bit you need to replace: 'tailpress.test\'

**Reply by BWBama85** [reply]
How can I change the port that is used? I already have a development enviroment and I just want it to watch on port 8080

**Reply by matoni109** [reply]
`--port \"5000\"`

**Comment by luki-zz** [comment]
Is brower-sync started with npm run watch? Or I need to start it seperatly?

**Comment by pguardiario** [comment]
Can someone update this answer please?

**Reply by charlie-george-1989** [reply]
Hey @pguardiario if you need help we can jump on a Zoom if you like

**Comment by pguardiario** [comment]
I don't think it's necessay, but thanks. I was wondering oc there was a hot
reload option but the tailwind play website is probably my next bet. Thanks
again.
